scripts/start.js

```diff
@@ -350,33 +338,28 @@ function getUserInfo (protocol, port) {
       if (err) {
         return reject(err)
       }
-      if (body['access_token']) {
-        const usersApiUrl = `${engineUrl}/api/users`
-        request(usersApiUrl, { json: true, strictSSL: false, headers: { Authorization: `Bearer ${body['access_token']}` } }, (userErr, userResponse, userBody) => {
-          // This request not always return current user data, it may be caused by server error, or current user isn't administrator and have no permissions for that
-          if (!userErr && userBody.user && userBody.user.length > 0 ) {
-            for (let i in userBody.user) {
-              if (userBody.user[i]['user_name'] === (username.slice(0, username.indexOf('@')) + '@' + domain)) {
-                resolve({
-                  userName: username.slice(0, username.indexOf('@')),
-                  ssoToken: body.access_token,
-                  domain: domain,
-                  userId: userBody.user[i].id,
+      if (body.access_token) {
+        const usersApiUrl = `${engineUrl}/api/`
+        request(usersApiUrl,
+                { json: true, strictSSL: false, headers: { Authorization: `Bearer ${body.access_token}` } }, 
+                (error1, response1, body1) => {
```

Comment:
How about `apiRootError, apiRootResponse, apiRootBody` or something else more descriptive?

Reply:
Gotcha.  Amended the commit with much better var names.